spec/brcobranca/retorno/cnab240/base_spec.rb

```diff
@@ -0,0 +1,23 @@
+# -*_ encoding: utf-8 -*-
+require 'spec_helper'
+
+RSpec.describe Brcobranca::Retorno::Cnab240::Base do
+  let(:arquivo) { File.join(File.dirname(__FILE__), '..', '..', '..', 'arquivos', nome_arquivo) }
```

Comment:
Line is too long. [97/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.